content/en/docs/Kyverno CLI/_index.md

```diff
@@ -224,3 +224,159 @@ policies:
 ```
+
+#### Policy Report
+
+Use '--policy_report' with apply command to generate policy report.
+
+Input combination:
+
+| S.No | policy        | resource         | cluster   | namespace      | interpretation                                                                           |
+| ---- |:-------------:| :---------------:| :--------:| :-------------:| :----------------------------------------------------------------------------------------| 
+| 1.   | policy.yaml   | -r resource.yaml | false     |                | apply policy from file to the resource from file                                         |
+| 2.   | policy.yaml   | -r resourceName  | true      |                | apply policy from file to the resource in cluster                                        |
+| 3.   | policy.yaml   |                  | true      |                | apply policy from file to all the resources in cluster                                   |
+| 4.   | policy.yaml   | -r resourceName  | true      | -n=namespace   | apply policy from file to the resource in cluster in mentioned namespace                 |
+| 5.   | policy.yaml   |                  | true      | -n=namespace   | apply policy from file to all the resources in cluster in mentioned namespace            |
+
+
+Example:
+
+Consider the following policy and resources:
+
+policy.yaml
+```yaml
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: require-pod-requests-limits
+  annotations:
+    policies.kyverno.io/category: Workload Management
+    policies.kyverno.io/description: >-
+      As application workloads share cluster resources, it is important to limit resources
+      requested and consumed by each pod. It is recommended to require 'resources.requests'
+      and 'resources.limits' per pod. If a namespace level request or limit is specified,
+      defaults are automatically applied to each pod based on the 'LimitRange' configuration.
+spec:
+  validationFailureAction: audit
+  rules:
+  - name: validate-resources
+    match:
+      resources:
+        kinds:
+        - Pod
+    validate:
+      message: "CPU and memory resource requests and limits are required"
+      pattern:
+        spec:
+          containers:
+          - resources:
+              requests:
+                memory: "?*"
+                cpu: "?*"
+              limits:
+                memory: "?*"
+```
+
+resource1.yaml
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx1
+  labels:
+    env: test
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+    imagePullPolicy: IfNotPresent
+    resources:
+      requests:
+        memory: "64Mi"
+        cpu: "250m"
+      limits:
+        memory: "128Mi"
+        cpu: "500m"
+```
+
+resource2.yaml
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx2
+  labels:
+    env: test
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+    imagePullPolicy: IfNotPresent
+```
+
+Case 1: Apply policy manifest on resource manifest
+
+```
```

Comment:
Fenced code blocks should declare a syntax method so the linter doesn't complain. Suggest to use "sh" for these.